.github/workflows/build-images.yaml

```diff
@@ -221,7 +223,7 @@ jobs:
         with:
           image: ${{ env.OPERATOR_NAME }}-catalog
           tags: ${{ env.IMG_TAGS }}
-          platforms: linux/amd64,linux/arm64
+          platforms: linux/${{ matrix.arch }}
```

Comment:
I wonder if by doing this we'd be executing the step multiple times, once for each arch.

Reply:
...and overwriting the same image. Fixed it with explicit tag names